src/lib/selector.c

```diff
@@ -273,6 +273,8 @@ ncselector* ncselector_create(ncplane* nc, int y, int x, const ncselector_option
 }
 
 int ncselector_additem(ncselector* n, const struct ncselector_item* item){
+  int origdimy, origdimx;
+  ncselector_dim_yx(n->ncp->nc, n, &origdimy, &origdimx);
```

Comment:
I should probably just trim this first argument from `ncselector_dim_yx()`, and access it therein through the `ncselector`. You're welcome to make this change if you'd like, but I will approve without it.